doc/man7/life_cycle-rand.pod

```diff
@@ -94,8 +94,8 @@ This is the canonical list.
  EVP_RAND_CTX_free          freed   freed      freed          freed
  EVP_RAND_get_ctx_params            newed   instantiated  uninstantiated  freed
  EVP_RAND_set_ctx_params            newed   instantiated  uninstantiated  freed
- EVP_RAND_CTX_gettable_params       newed   instantiated  uninstantiated  freed
- EVP_RAND_CTX_settable_params       newed   instantiated  uninstantiated  freed
+ EVP_RAND_CTX_get_params            newed   instantiated  uninstantiated  freed
+ EVP_RAND_CTX_set_params            newed   instantiated  uninstantiated  freed
```

Comment:
The original was correct. And yeah - it is awkward as the EVP_RAND_get/set_ctx_params() does not follow the naming pattern from EVP_MAC and EVP_KDF. But that rename is for a different PR.